kedro_kubeflow/vertex_ai/io.py

```diff
@@ -54,6 +54,12 @@ def is_file_path_input(input_data):
     return input_params, input_specs
 
 
+def get_output_type(output, catalog):
+    if catalog[output].get("layer") == "models":
```

Comment:
Am I correct, that this fragment picks also mlflow artifacts if these are stored locally? How does it work if artifacts are stored in remote mlflow server?

Reply:
That's right, this part does not differentiate between remote and local artifacts. Unfortunately I wasn't able to test the mlflow artifacts yet due to connectivity issues, but as soon as I overcome these, I will update the code to support these somehow